.github/workflows/release.yml

```
@@ -0,0 +1,112 @@
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)